HHS/Head-Start-TTADP
# [TTAHUB-447] Better messaging in the case of export failures

## Description of change
Download failures should be try/catchable, so that an error, even one that breaks the backend, will display a meaningful message to the users. 

## How to test
Attempt a download that triggers an error. Observe that an error message is displayed. (This works better in environments with redundant instances)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-447

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [/na] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
